patch/llpcPatchPeepholeOpt.h

```diff
@@ -93,6 +101,7 @@ class PatchPeepholeOpt final:
     // -----------------------------------------------------------------------------------------------------------------
 
     llvm::SmallVector<llvm::Instruction*, 8> m_instsToErase;
+    bool m_enKillOpt;
```

Comment:
m_enKillOpt -> m_enableDiscardOpt; // Whether to enable the optimization for "kill" intrinsic

Add the missing comments